Compfiles/Imo1959P4.lean

```diff
@@ -0,0 +1,21 @@
+/-
+Copyright (c) 2024 The Compfiles Contributors. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: InternLM-MATH LEAN Formalizer v0.1. This is auto-formalized by InternLM-MATH LEAN Formalizer v0.1, modified and verified by InternLM-MATH team members.
```

Comment:
The `Authors` line should be a comma-separated list of authors and not include extra explanations. We parse this list for display on the compfiles web page: https://github.com/dwrensha/compfiles/blob/f99bf6f2928d47dd1a445b414b3a723c2665f091/ProblemExtraction.lean#L133-L139

Just "InternLM-MATH LEAN Formalizer v0.1" would be OK.